docs/k8s/README.md

```diff
@@ -16,7 +16,7 @@ There are quite a few ways of running Tailscale inside a Kubernetes Cluster, som
    metadata:
      name: tailscale-auth
    stringData:
-     TS_AUTH_KEY: tskey-...
+     AUTH_KEY: tskey-...
```

Comment:
run.sh uses TS_AUTH_KEY: https://github.com/tailscale/tailscale/blob/027111fb5a3785c4cc3a68d8ad347eece380e9e7/docs/k8s/run.sh#L9

sidecar.yaml needs to be updated to use TS_AUTH_KEY instead.

Reply:
Fixed: e7476e2dd66151590695bae9018d5e3c51e089e5